### PR TITLE
Fixed android warnings

### PR DESCRIPTION
**Pull request by Losev-Ivan**

### CODE REVIEW

File: _studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp [hunk 1]
```diff
@@ -629,7 +629,11 @@ mfxStatus ImplementationAvc::QueryIOSurf(
         checkSts = lpSts;
 
     SetDefaults(tmp, hwCaps, true, core->GetHWType(), core->GetVAType(), *QueryCoreInterface<eMFXGTConfig>(core, MFXICORE_GT_CONFIG_GUID));
+
+#ifdef MFX_VA_WIN
     mfxExtCodingOption3 const &   extOpt3 = GetExtBufferRef(tmp);
```
Comment (dvrogozh) [inline _studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp]: extOpt3 is not used: there is no path for MFX_VA_WIN which uses it. Just remove.

**Comment by Losev-Ivan** [comment]
I made this patch considering the changes that @akokoshn made.

**Comment by dvrogozh** [comment]
In general I am ok with the change. @akokoshn : you indeed did not fix that already:)?

File: _studio/mfx_lib/decode/vp8/include/mfx_vp8_dec_decode_vp8_defs.h [hunk 2]
```diff
@@ -127,10 +127,6 @@ enum
 
 extern const uint8_t  vp8_range_normalization_shift[64];
 
-extern const int32_t vp8_quant_dc[VP8_MAX_QP + 1 + 32];
```
Comment (akokoshn) [inline _studio/mfx_lib/decode/vp8/include/mfx_vp8_dec_decode_vp8_defs.h]: Already done in PR: https://github.com/Intel-Media-SDK/MediaSDK/pull/455

File: _studio/mfx_lib/decode/vp8/src/mfx_vp8_dec_decode_hw.cpp [hunk 3]
```diff
@@ -926,89 +926,6 @@ void VideoDECODEVP8_HW::UpdateLoopFilterDeltas(MFX_VP8_BoolDecoder &dec)
   } \
 }
 
-static const int32_t vp8_quant_ac[VP8_MAX_QP + 1 + 32] =
```
Comment (akokoshn) [inline _studio/mfx_lib/decode/vp8/src/mfx_vp8_dec_decode_hw.cpp]: Already done in PR: https://github.com/Intel-Media-SDK/MediaSDK/pull/455

File: _studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp [hunk 4]
```diff
@@ -629,7 +629,6 @@ mfxStatus ImplementationAvc::QueryIOSurf(
         checkSts = lpSts;
 
     SetDefaults(tmp, hwCaps, true, core->GetHWType(), core->GetVAType(), *QueryCoreInterface<eMFXGTConfig>(core, MFXICORE_GT_CONFIG_GUID));
-    mfxExtCodingOption3 const &   extOpt3 = GetExtBufferRef(tmp);
```
Comment (akokoshn) [inline _studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp]: Already done in PR: https://github.com/Intel-Media-SDK/MediaSDK/pull/455

File: _studio/shared/asc/src/motion_estimation_engine.cpp [hunk 5]
```diff
@@ -215,21 +215,6 @@ void MVpropagationCheck(mfxI32 xLoc, mfxI32 yLoc, ASCImDetails dataIn, ASCMVecto
 
 #define SAD_SEARCH_VSTEP 2  // 1=FS 2=FHS
 
-alignas(16) static const mfxU16 tab_twostep[8] = {
```
Comment (akokoshn) [inline _studio/shared/asc/src/motion_estimation_engine.cpp]: Already done in PR: https://github.com/Intel-Media-SDK/MediaSDK/pull/455

File: _studio/shared/src/mfx_vpp_vaapi.cpp [hunk 6]
```diff
@@ -2273,8 +2273,6 @@ mfxStatus VAAPIVideoProcessing::Execute_Composition(mfxExecuteParams *pParams)
 
 mfxStatus VAAPIVideoProcessing::QueryTaskStatus(mfxU32 taskIndex)
 {
-    VAStatus vaSts;
```
Comment (akokoshn) [inline _studio/shared/src/mfx_vpp_vaapi.cpp]: Already done in PR: https://github.com/Intel-Media-SDK/MediaSDK/pull/455

Comment (akokoshn) [reply _studio/shared/src/mfx_vpp_vaapi.cpp]: So could you please update your local repo and do manually merge

**Comment by Losev-Ivan** [comment]
Fixed patch according to @akokoshn  changes in PR [455](https://github.com/Intel-Media-SDK/MediaSDK/pull/455)

**Comment by Losev-Ivan** [comment]
This PR was separated on 3:
https://github.com/Intel-Media-SDK/MediaSDK/pull/609
https://github.com/Intel-Media-SDK/MediaSDK/pull/611
https://github.com/Intel-Media-SDK/MediaSDK/pull/612

**Pull request closed without merging**